platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -25,6 +25,8 @@ android {
         minSdkVersion androidVersions.minSdkVersion
         targetSdkVersion androidVersions.targetSdkVersion
         buildConfigField "String", "GIT_REVISION_SHORT", String.format("\"%s\"", getGitRevision())
+        buildConfigField "String", "MAPBOX_SDK_IDENTIFIER", String.format("\"%s\"", "mapbox-maps-android")
+        buildConfigField "String", "MAPBOX_SDK_VERSION", String.format("\"%s\"", project.VERSION_NAME)
         buildConfigField "String", "MAPBOX_VERSION_STRING", String.format("\"Mapbox/%s\"", project.VERSION_NAME)
```

Comment:
Is `MAPBOX_VERSION_STRING` still being used or was it replaced by `MAPBOX_SDK_VERSION`?

Reply:
It's still being used in [`HTTPRequest`](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HTTPRequest.java) 👀 https://github.com/mapbox/mapbox-gl-native/blob/e2a4dffcb627498561d66a304ec668f8242b8342/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HTTPRequest.java#L210